initializer.py

```diff
@@ -0,0 +1,231 @@
+
```

Comment:
I suggest excluding this file for now. We can focus on the rewriter rules for this PR.

Reply:
Got it